python/pynessie/cli.py

```diff
@@ -465,7 +466,8 @@ def _get_contents(nessie: NessieClient, ref: str, delete: bool = False, *keys: s
                 + " Removing the content results in a delete"
             )
             try:
-                message = click.edit(edit_message)
```

Comment:
What is hte reason to change from `None` to empty string in the return type?

Reply:
it's been "enforced" by flake8 (`None` isn't compatible to `str`)

Reply:
No clue, why that didn't happen earlier though

Reply:
This will change behaviour. This will (as far as I can tell) allow empty commit messages. I don't think this is correct, though I am not 100% convinced it was correct originally.